### PR TITLE
Single `/api/registration` for everything

### PR DESCRIPTION
**Pull request by gabriel-v**
According to https://github.com/liquidinvestigations/setup/issues/52

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -8,16 +8,103 @@ API allowing for Liquid Investigations system management
...
+                "netmask": "255.255.255.0",
+                "dhcp_range": "10.0.0.100-255",
+                "hotspot": {
+                    "ssid": "123b456x",
+                    "password": "123b456x"
+                },
+                "eth": false
+            },
+            "wan": {
+                "wifi": {
```
Comment (mgax) [inline apiary.apib:98]: We can't return any ssid+password because our database is in a blank state.

File: apiary.apib [hunk 2]
```diff
@@ -8,16 +8,103 @@ API allowing for Liquid Investigations system management
...
             "username": "johnDoe",
-            "password": "123b456x"
+            "password": "123b456x",
+            "domain": "liquidnode.liquid",
+            "lan": {
+                "ip": "10.0.0.1",
+                "netmask": "255.255.255.0",
+                "dhcp_range": "10.0.0.100-255",
+                "hotspot": {
+                    "ssid": "123b456x",
```
Comment (mgax) [inline apiary.apib:92]: We can't provide a default value for hotspot ssid+password, it should be up to the user, with no defaults.

Comment (gabriel-v) [reply apiary.apib:92]: You're looking at the POST section. This is an example of what the user fills in. You can see it says `+ Request`, not `+ Response`. Scroll up for the GET section.

File: apiary.apib (hunk 1 above)
Comment (gabriel-v) [reply apiary.apib:98]: Same as above. This is what the user fills in, not what we're returning.
The GET returns blanks for ssid+passwords.

**Pull request merged**